server/package.json

```diff
@@ -12,7 +12,7 @@
     "test_debug": "npm run test -- --inspect-brk",
     "serverless": "serverless offline start",
     "serverless:print": "serverless print",
-    "debug": "export SLS_DEBUG=* && node --inspect-brk ./node_modules/.bin/serverless offline -s dev"
+    "debug": "set SLS_DEBUG=* && node --inspect /usr/local/lib/node_modules/serverless/bin/serverless offline -s dev"
```

Comment:
This is not where my serverless is installed :-(

Reply:
Where is your serverless installed?

Reply:
/use/bin/serverless

Reply:
Switched to using local one. It seems to work. Maybe at some point in the past you had to use the globally installed one.